lib/fog/brightbox.rb

```diff
@@ -1,46 +0,0 @@
-require "fog/core"
-require "fog/json"
-
-# See the main fog gem for more details about the top level class {Fog}
```

Comment:
Just wondering. Wont this break?
AFAIK bundler will try to load the path `fog/brightbox` when this gem is listed in a `Gemfile` and thus will raise an exception.